### PR TITLE
[OBSDEF-1221]Add charts for Pravega deployment in ObjectScale

### PR DESCRIPTION
**Pull request by AaronCOD**
## Purpose
[OBSDEF-1221](https://jira.cec.lab.emc.com:8443/browse/OBSDEF-1221)
add image repo for Pravega components, add enable Pravega flag.

## PR checklist
- [x] make test passed
- [ ] make build passed
- [ ] helm install <chart> passed
- [ ] vSphere7 make package deployments passed
- [ ] helm upgrade <chart> passed
- [ ] helm test <release_name> passed

## Testing
_Paste the output of your helm install/vSphere7 deployment testing here_

### CODE REVIEW

**Review by eddiepavkovic — DISMISSED** [review]

File: pravega-operator/values.yaml [hunk 1]
```diff
@@ -9,8 +9,8 @@ global:
   watchAllNamespaces: true
 
 image:
-  repository: pravega/pravega-operator
-  tag: 0.5.1
+  repository: devops-repo.isus.emc.com:8116/nautilus/pravega-operator
```
Comment (fedor-chemashkin) [inline pravega-operator/values.yaml:12]: We can't put here our internal repositories. We can override them in tests with the necessary values.

File: objectscale-manager/crds/pravega-crd.yaml [hunk 2]
```diff
@@ -1,806 +0,0 @@
----
```
Comment (fedor-chemashkin) [inline objectscale-manager/crds/pravega-crd.yaml:1]: Needs to be here for correct work. Pravega operator chart should be updated to support Helm 3

File: ecs-cluster/values.yaml [hunk 3]
```diff
@@ -100,24 +102,57 @@ atlas:
 pravega:
   # labels:
   image:
-    repository: pravega
-  # tag: ...
+    repository: devops-repo.isus.emc.com:8116/nautilus/pravega
+    tag: 0.9.0-2732.066a3ad0b-SLTS_FOR_FLEX_12_14
```
Comment (fedor-chemashkin) [inline ecs-cluster/values.yaml:106]: We can't put here our internal repositories. We can override them in tests with the necessary values.

File: bookkeeper-operator/values.yaml [hunk 4]
```diff
@@ -9,9 +9,9 @@ global:
   watchAllNamespaces: true
 
 image:
-  repository: pravega/bookkeeper-operator
-  tag: 0.1.2
-  pullPolicy: IfNotPresent
+  repository: asdrepo.isus.emc.com:8099/pravega/bookkeeper-operator
```
Comment (fedor-chemashkin) [inline bookkeeper-operator/values.yaml:12]: We can't put here our internal repositories. We can override them in tests with the necessary values.

File: bookkeeper-operator/values.yaml [hunk 5]
```diff
@@ -9,9 +9,9 @@ global:
...
+  tag: v0.1.3
+  pullPolicy: Always
```
Comment (fedor-chemashkin) [inline bookkeeper-operator/values.yaml:14]: Pull policy should be `IfNotPresent`. It can be overridden

**Review by alexander-rassanov — APPROVED** [review]

File: ecs-cluster/values.yaml [hunk 6]
```diff
@@ -96,31 +98,61 @@ atlas:
   persistence:
     # Size of each volume to be provisioned
     size: 32Gi
-
     # Name of the storage class that storage should be provisioned from
     # storageClassName:
-
+pravegaRepository: devops-repo.isus.emc.com:8116/nautilus
```
[the remark this answers is not in the document]
Comment (AaronCOD) [reply ecs-cluster/values.yaml:103]: It is a temp solution since we do not have all images centralized in one place.

**Review by fedor-chemashkin — APPROVED** [review]

**Pull request merged**